pkg/cmd/pulumi/new.go

```diff
@@ -535,6 +578,17 @@ func newNewCmd() *cobra.Command {
 	cmd.PersistentFlags().BoolVarP(
 		&args.listTemplates, "list-templates", "l", false,
 		"List locally installed templates and exit")
+	cmd.PersistentFlags().StringVar(
+		&args.aiPrompt, "ai", "", "Prompt to use for Pulumi AI",
+	)
+	cmd.PersistentFlags().Var(
+		&args.aiLanguage, "language", "Language to use for Pulumi AI "+
+			"(possible choices: TypeScript, JavaScript, Python, Go, C#)",
```

Comment:
Need to update this for YAML and Java.

Reply:
We've omitted these for now as the code quality is not at the level we feel comfortable with.

Reply:
On reflection, I'll update the PR to add these languages and let issues filed so that we get user requests to drive specific improvement here.